view.php

```diff
@@ -23,15 +23,42 @@
  * @link http://www.mantisbt.org
  *
  * @uses core.php
+ * @uses access_api.php
+ * @uses authentication_api.php
+ * @uses gpc_api.php
  */
 
 require_once( 'core.php' );
+require_api( 'access_api.php' );
+require_api( 'authentication_api.php' );
+require_api( 'gpc_api.php' );
 
 $t_file = __FILE__;
 $t_mantis_dir = dirname( __FILE__ ) . DIRECTORY_SEPARATOR;
 $t_show_page_header = true;
 $t_force_readonly = false;
 $t_fields_config_option = 'bug_view_page_fields';
+$f_bug_id = gpc_get_int( 'id' );
+if( ( ON === config_get_global( 'public_urls' ) ) ) {
+	$f_bug_token = gpc_get_string( 'token', '' );
+	$f_full_view = false;
```

Comment:
$f_ is for form variables.  In this case, bug_token and full_view are really temp variables that should use $t_ prefix.  If you want the variables exposed to included scripts below, technically speaking I would use $g_, but since $t_ is used for others, you can go either way.